### ISSUE TITLE
dzsave corrupts zips > 4 GB

### ISSUE THREAD

**Issue by bgilbert** [jcupitt/libvips#138]
```shell
$ vips dzsave CMU-1.mrxs out2.zip --suffix=.png
$ unzip out2.zip | grep -v extracting
warning [out2.zip]:  4294967296 extra bytes at beginning or within zipfile
  (attempting to process anyway)
Archive:  out2.zip
file #1:  bad zipfile offset (local header sig):  4294967296
  (attempting to re-compensate)
file #418485:  bad zipfile offset (local header sig):  61544
  (attempting to re-compensate)
$ ls -l out2.zip
-rw-rw-r--. 1 bgilbert bgilbert 4391950384 Jun 25 04:05 out2.zip
```

It turns out that libgsf [doesn't support ZIP64 extensions](https://bugzilla.gnome.org/732209) and produces corrupt archives when ZIP64 would be necessary.

**Comment by bgilbert** [comment]
ZIP64 should be available in libgsf >= 1.14.31.

**Comment by jcupitt** [comment]
This patch removes the limit for recent libgsf: https://github.com/jcupitt/libvips/commit/33788f7336537a8c65c3b07973a59f6fbf82c4ab

Thanks @bgilbert !
